### PR TITLE
Upgrade to Electron 3

### PR DESCRIPTION
**Pull request by sindresorhus**
Electron 3 comes with newer Node.js version, so we can use object spread in the main process now too.

I've been playing with it for a while and everything seems to be working.

### CODE REVIEW

File: main/share-service-context.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ class ShareServiceContext {
       return;
     }
 
-    const request = got(url, Object.assign({}, options, {useElectronNet: false}));
+    const request = got(url, options);
```
Comment (sindresorhus) [inline main/share-service-context.js]: `useElectronNet: false` is the default already in the got version we're using.

File: main/export.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ class Export {
           });
         }
         resolve();
-      } catch (error) {
+      } catch (_) {
         reject();
```
Comment (albinekb) [inline main/export.js]: Why not reject with the error here? 🤔

Comment (sindresorhus) [reply main/export.js]: I was going to ask @karaggeorge about that. I changed it to `_` to make it clear that the error is ignored.

Comment (albinekb) [reply main/export.js]: Yep, better for another PR perhaps

Comment (karaggeorge) [reply main/export.js]: Hmm, I don't remember. Probably should `reject(error)` but yeah we can leave that for a separate PR if you want

**Pull request merged**